### PR TITLE
[CNFT2-544] - Create API to pull question bank from PHIN VADS api

### PR DESCRIPTION
**Pull request by lramakr**
- The PHIN VADS API integrated into the question-bank micro service to allow searching for standard PHIN value sets.
- https://cdc-nbs.atlassian.net/browse/CNFT2-544
Screenshot 2023-06-28 at 8 47 08 AM

Screenshot 2023-06-28 at 8 47 22 AM

### CODE REVIEW

**Comment by mpeels** [comment]
Is there a way to query PHIN VADS using a value set or question name instead of just the oid?

**Comment by lramakr** [comment]
> Is there a way to query PHIN VADS using a value set or question name instead of just the oid?

Yes Mike. We can use this way as well
`https://phinvads.cdc.gov/baseStu3/ValueSet/PHVS_AdministrativeGender_HL7_V3`

File: gradle/libs.versions.toml [hunk 1]
```diff
@@ -43,6 +44,7 @@ cucumber-junit-platform-engine = { module = "io.cucumber:cucumber-junit-platform
 junit = { module = "org.junit.jupiter:junit-jupiter" }
 junit-platform = { module = "org.junit.platform:junit-platform-suite" }
 awaitility = {module = "org.awaitility:awaitility"}
+validation-api = {module = "javax.validation:validation-api", version.ref = "validation"}
```
Comment (mpeels) [inline gradle/libs.versions.toml:47]: Is this being used anywhere?

Comment (lramakr) [reply gradle/libs.versions.toml]: Remove the changes

**Review by mpeels — APPROVED** [review]

**Pull request merged**